assignments/arrays.js

```diff
@@ -72,25 +78,48 @@ console.log(`Car 33 is a *car year goes here* *car make goes here* *car model go
 let lastCar = 0;
 console.log();
 
+let lastCar = inventory[inventory.length - 1];
```

Comment:
Remember that a let variable cannot be declared twice so you would just want lastCar here without adding another let to it.